packages/lib/src/artifacts/ZWeb3.ts

```diff
@@ -50,6 +50,10 @@ export default class ZWeb3 {
     return Web3.utils.toWei(value, unit);
   }
 
+  public static fromWei(value: string, unit: any): string {
```

Comment:
I'd consider removing this and other utils functions from here. They are in zweb3 only because they are implemented by web3, but they are actually unrelated to the connection to the network. We can tackle this later, of course.